docs/man/decrypt/_index.md

```diff
@@ -40,3 +43,13 @@ Advanced piping is supported
 $ echo "hello world" | otdfctl encrypt | otdfctl decrypt | cat
 hello world
 ```
+
+Assertion verification:
```

Comment:
Can you please clarify in these docs a little more about the assertion feature for clarity:
- which flow expects a private key and which expects a public key (or if both expect both, which I don't believe is the case)
- a fully fleshed out example JSON with the expected format including `\n` breaks
- how decrypt in the CLI behaves with and without these keys provided if a TDF contains assertions